android/expoview/src/main/java/host/exp/exponent/notifications/managers/SchedulersManagerProxy.java

```diff
@@ -0,0 +1,64 @@
+package host.exp.exponent.notifications.managers;
+
+import android.content.Context;
+
+import org.unimodules.core.interfaces.Function;
+
+import java.util.concurrent.Executor;
+import java.util.concurrent.Executors;
+
+import host.exp.exponent.notifications.interfaces.SchedulerInterface;
+import host.exp.exponent.notifications.interfaces.SchedulersManagerInterface;
+
+public class SchedulersManagerProxy implements SchedulersManagerInterface {
+
+  private volatile static SchedulersManagerInterface instance = null;
```

Comment:
Why does it have to be a singleton like that? Couldn't it be a kernel service (created once per lifetime of the application, not relying on experiences)?

Reply:
To be changed when moving to unimodule — there we'll use unimodules' singleton.